recipes-bsp/u-boot/u-boot-iot2050_2021.04.inc

```diff
@@ -1,15 +1,17 @@
 #
-# Copyright (c) Siemens AG, 2020-2021
+# Copyright (c) Siemens AG, 2020-2022
 #
 # Authors:
 #  Jan Kiszka <jan.kiszka@siemens.com>
+#  Su Baocheng <baocheng.su@siemens.com>
 #
 # This file is subject to the terms and conditions of the MIT License.  See
 # COPYING.MIT file in the top-level directory.
 #
 
 require u-boot-iot2050.inc
 
+PV = "2021.04"
```

Comment:
Not needed. Will be derived from the including recipe.

Reply:
Seems needed, see: https://github.com/siemens/meta-iot2050/runs/4709773299?check_suite_focus=true

Reply:
Yes, missed that you didn't properly versioned the recipes ("..._2021.04"). Fix that, and this will work.

Reply:
I was thinking of add the PV to the filename of pg1.bb and pg2.bb, however, don't you think this seems tedious and kind of ugly to have triple `2021.04`?

Reply:
Nope, why? We do this for the kernel as well.